src/torrent.cpp

```diff
@@ -577,6 +577,14 @@ bool is_downloading_state(int const st)
 				do_connect_boost();
 			}
 
+			if (!is_ssl_torrent() && !p.root_certificate.empty())
```

Comment:
it's not obvious that the `is_ssl_torrent()` check is necessary or correct. Do you not want it to be possible to override a root certificate if there already exists one in the .torrent file?

Reply:
Yes that was my intention, but you're right it seems unnecessary.